src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -13,41 +13,124 @@ tags:
+### New Relic One user model
 
-To learn more about account/user models, see [User model overview](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models).
+If your organization is part of the [New Relic One pricing model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/), the user types, privileges, and capabilities are different than those of our original user model:
 
-## Account access
+| | Basic user | Full user |
+|---|---|---|
+| Serve Nerdpacks | yes | yes |
+| Publish Nerdpacks | no | yes |
+| Subscribe to Nerdpacks | no | yes |
+| Tag Nerdpacks | no | yes |
+| Use Nerdlets or visualizations created by the user's accounts | no | yes |
+| Use Nerdlets created by New Relic | no* | yes |
+| Use visualizations created by New Relic | no | yes |
 
-For organizations with master/sub-account structures:
+_* There are a few Nerdlets that basic users are allowed to use. See [Basic users](#basic-users-1) for more information._
 
-* If you subscribe to a Nerdpack from a master account, that access is inherited by all of its sub-accounts.
-* A Nerdpack made by your team can only be subscribed to from the master account that was used to publish it, or from its sub-accounts. This means that, if the Nerdpack needs to be available across your organization, you may need a New Relic admin to deploy it.
\ No newline at end of file
+## Nerdpack permissions: Original user model
+
+On both the [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/) and the [New Relic One user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/), user permissions differ between types and roles. In this section, you learn the differences between how basic users, full users, and admins can use Nerdpacks in the original user model.
+
+### Basic users
+
+Basic users in the original user model are limited in their Nerdpack capabilities. They can only create and serve Nerdpacks locally. To publish, tag, subscribe to, and use their Nerdpacks, a basic user must be upgraded to a full user. An admin user on their account can do this for them.
```

Comment:
Can you make this "you" centered - "If you're a basic user, you have limited Nerdpack capabilites. You can only create... "

Reply:
Yeah, I was wondering if this might be raised 😄 I thought about this a lot and the reason I opted against it was because it feels more like a strictly generic doc (similar to [this section](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/#user-type)) as the audience isn't 100% clear. For example, I might be the basic user who came across an issue, so I'm looking it up. Or, I might be the basic user's admin looking this up on their behalf. I'm probably over-thinking this and could go either way... so yes, I'll change it 👍